conf/deploy.json

```diff
@@ -6,6 +6,16 @@
             "data": {
                 "bucket": "content-api-dist"
             }
+        },
+        "content-api-dashboard-ami-update" : {
+            "type": "ami-cloudformation-parameter",
+            "data": {
+                "amiTags" : ["ubuntu-wily-java-8"],
```

Comment:
Thanks, I wasn't sure on this so just stuck it up. In your tags its referenced as `Recipe` rather than `RecipeId` ... does that need to change?